3001-4000/LIT3326Melchiz1.xml

```diff
@@ -4,7 +4,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">First homily by Cyril of Alexandria on Melchizedek</title>
+            <title xml:id="t1" type="normalised">First homily by Cyril of Alexandria on Melchizedek</title>
```

Comment:
I do not know why the schema does not signal the error to you but the accepted values only include `normalized `with z, we do not have `normalised `at all.

More importantly the meaning of `normalized `is `transcription`, the correct way to encode such title is either 

```
<title xml:id="t1" xml:lang="en">First homily by Cyril of Alexandria on Melchizedek</title>
<title xml:lang="gez" xml:id="t2"> ድርሳን፡ ዘቄርሎስ፡ ቅዱስ፡ ሊቀ፡ ጳጳሳት፡ ዘአለስክንድርያ፡ በእንተ፡ መልከ፡ ጼዴቅ፡</title>
```

that is title 1 English traditional title 2 Ethiopic

or 

```
<title xml:lang="gez" xml:id="t1"> ድርሳን፡ ዘቄርሎስ፡ ቅዱስ፡ ሊቀ፡ ጳጳሳት፡ ዘአለስክንድርያ፡ በእንተ፡ መልከ፡ ጼዴቅ፡</title>
<title xml:lang="gez" type="normalized" corresp="#t1">Dersan etc etc (the transcription of Ethiopic)</title>
<title xml:lang="en" corresp="#t1">Homily by Saint Cyril the Patriarch of Aleksandria on Melchisedek (meaning the literal English translation of the Ethiopic title)</title>
```

then it will be printed as Fidal (transcription, 'translation')

If you want all grouped as `t1 `but with the English printed first (preferable for non-Ethiopic works) then you can add `type="main" `to the English version

Please check the guidelines for encoding titles